### ISSUE TITLE
UNSUBSTANTIAL alerted on Azure links

### ISSUE THREAD

**Issue by rsese** [atom/biohazard-alert#10]
Jason dropped a few CI link comments and we got 3 alerts about them - example comment from https://github.com/atom/atom/issues/18993#issuecomment-479519315:

```
Another one:

https://github.visualstudio.com/Atom/_build/results?buildId=37069&view=logs&jobId=a5e52b91-c83f-5429-4a68-c246fc63a4f7&taskId=8b9bff53-2841-58c7-68fc-e30c30330fc3&lineStart=181&lineEnd=183&colStart=1&colEnd=1
```

From the alert:

```
UNSUBSTANTIAL: 0.866102
```

**Comment by lee-dohm** [comment]
It looks like 0.8 is not a sufficient threshold for what we would typically consider a "too short" response under the UNSUBSTANTIAL model. We may need to bump it to 0.9. That will require a code change that allows for different thresholds for different models, however.

**Comment by lee-dohm** [comment]
Fixed in #14.
